rskj-core/src/main/java/co/rsk/rpc/ExecutionBlockRetriever.java

```diff
@@ -69,27 +66,26 @@ public BlockResult retrieveExecutionBlock(String bnOrId) {
         }
 
         if (PENDING_ID.equals(bnOrId)) {
```

Comment:
Minor thing, but I think this logic should be moved to its own method for the sake of readability.

Reply:
agree, done